Dockerfile

```diff
@@ -33,6 +34,9 @@ RUN	curl -L -o /bin/kops https://github.com/kubernetes/kops/releases/download/$K
 RUN	curl -L -o- https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar -zxvf - && cp linux-amd64/helm \
 	/bin/helm && chmod 755 /bin/helm && helm init --client-only
 
+RUN curl -L -o- https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_VERSION.tgz | tar -zxvf - && cp docker/docker \
```

Comment:
We no longer depend on mounting the docker executable from the underlying system, as there's no guarantee that it will be statically linked (and therefore run inside our alpine/musl-based aladdin container).  Since we already directly download the docker executable in infra_k8s_check.sh, doing it here too seemed reasonable.